Geometry/CSCGeometryBuilder/test/python/validateCSCGeometry2_cfg.py

```diff
@@ -0,0 +1,53 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
Call this file `validateCSCGeometryOldDD_cfg.py` or something like that.

Reply:
Done